spec/unit/providers/gce_spec.rb

```diff
@@ -94,7 +101,6 @@
         result = subject.create_vm(poolname, vmname)
         #result = subject.destroy_vm(poolname, vmname)
         # subject.get_vm(poolname, vmname)
-        subject.dns_teardown({'name' => vmname})
         # subject.dns_setup({'name' => vmname, 'ip' => '1.2.3.5'})
```

Comment:
Verify the commented out bits still need to be here for use when manually testing things.

Reply:
Removed those since the setup and teardown in done in vmpooler core now.